tests/preprocessing/test_positionfixes.py

```diff
@@ -23,6 +23,20 @@ def geolife_pfs_stps_long():
 class TestGenerate_staypoints:
     """Tests for generate_staypoints() method."""
 
+    def test_duplicate_columns(self):
+        """Test if running the function twice, the generated column does not yield exception in join statement"""
+
+        # we run generate_staypoints twice in order to check that the extra column(tripleg_id) does
+        # not cause any problems in the second run
+        pfs, _ = ti.io.dataset_reader.read_geolife(os.path.join("tests", "data", "geolife"))
+        pfs_run_1, stps = pfs.as_positionfixes.generate_staypoints(
+            method="sliding", dist_threshold=0, time_threshold=0, include_last=True
```

Comment:
Here may be using a "normal" parameter set would be better. Like ` dist_threshold=100, time_threshold=5.0`

Reply:
done